sample-apps/discount-functions-sample-app/extensions/order-discount/src/main.rs

```diff
@@ -66,18 +32,14 @@ fn function(input: input::Input) -> Result<FunctionResult, Box<dyn std::error::E
 mod tests {
     use super::*;
 
-    fn input(configuration: Option<Configuration>) -> input::Input {
-        let default_input: input::Input = serde_json::from_str("{}").unwrap();
-        let discount_node = Some(input::DiscountNode {
-            metafield: Some(input::Metafield {
-                value: serde_json::to_string(&configuration).ok(),
-            }),
-        });
+    fn input(configuration: Option<input::Configuration>) -> input::Input {
+        let discount_node = input::DiscountNode {
+            metafield: input::Metafield {
+                value: serde_json::to_string(&configuration).ok().unwrap(),
```

Comment:
I think the `.ok()` is unnecessary. We can just unwrap the `Result` instead of converting to an `Option` and then unwrapping